### DISCUSSION TITLE
More clarity in the readme file

### DISCUSSION THREAD

**Discussion by cpach** [abiosoft/colima#181] [Ideas]
Please consider adding some additional clarification and context in the readme file.

To me as an outsider, reading the readme didn’t really help me to better understand the distinction between colima and lima, and the purpose that colima serves.

I guess that one selling point for colima is that it makes it easier to have port-forwarding and bind mounts out of the box…? If so, it might be a good idea to add more information about that.

Just my 2¢.

**Comment by cpach** [comment]
Hi folks! No comments yet, so I’m wondering if anyone please could chime in and share your thoughts about this?

**Comment by abiosoft** [comment]
The FAQ was updated to include some info in that regards https://github.com/abiosoft/colima#faq

**Reply by cpach** [reply]
That’s great. Very good description there IMHO. Thank you @abiosoft!
